### PR TITLE
Add first working prototype of QC on TOF digits

### PR DESCRIPTION
**Pull request by njacazio**
Checking:
- hit multiplicity
- hit time
- hit ToT

- Improve checker with useful messages
- Made limits (hard) in the header file -> to be switched to configurable

### CODE REVIEW

**Comment by njacazio** [comment]
Ciao @ktf, I think my PR build is stuck, could you please have a look?

**Comment by ktf** [comment]
@MichaelLettrich can you take a look? Thanks.

**Comment by MichaelLettrich** [comment]
the macos builder died. restarted and should be up and running soon.

**Comment by ktf** [comment]
can you restart the linux one as well?

File: Modules/TOF/src/TOFCheckRawsMulti.cxx [hunk 1]
```diff
@@ -80,73 +81,79 @@ Quality TOFCheckRawsMulti::check(std::map<std::string, std::shared_ptr<MonitorOb
...
-              isINT7AverageHigh = kTRUE;
+          } else { // Running with collisions
+            const Bool_t isZeroBinContentHigh = (zeroBinIntegral > fracAtZeroMult * totIntegral);
+            const Bool_t isLowMultContentHigh = (lowMIntegral > fracAtLowMult * totIntegral);
+            const Bool_t isINT7AverageLow = (multiMean < minTOFrawhits);
+            const Bool_t isINT7AverageHigh = (multiMean > maxTOFrawhits);
 
             // if (AliRecoParam::ConvertIndex(specie) == AliRecoParam::kLowMult) {
-            if (0) {
+            if (0) { // TODO: Low multiplicity running, how to check if it is pp? Probably this can be simplified in the json
```
Comment (knopers8) [inline Modules/TOF/src/TOFCheckRawsMulti.cxx:119]: Good question. One way would be to add a parameter to a Check json configuration:
```
"MyCheck": {
  ...
  "checkParameters" : {
    "runType" : "cosmics"
  }
 ...
}
```
But in a longer run, I think we should consider adding this kind of metadata to each MonitorObject in framework, as soon as we are able to retrieve it. @Barthelemy what do you think?

Comment (njacazio) [reply Modules/TOF/src/TOFCheckRawsMulti.cxx:119]: I was thinking of removing all the ifs from the task and simply feeding it different json files depending on the running conditions so as to entirely delegate this to the initial configuration but  suggestions are most welcome!

**Pull request merged**